lib/api/core/plugins/pluginContext.js

```diff
@@ -135,19 +136,46 @@ class PluginContext {
         get: () => curryTrigger(kuzzle, pluginName)
       });
 
+      const getPluginSdk = sdk => ({
+        query: sdk.query.bind(sdk),
+        auth: sdk.auth,
+        bulk: sdk.bulk,
+        collection: sdk.collection,
+        document: sdk.document,
+        index: sdk.index,
+        ms: sdk.ms,
+        realtime: () => {
+          throw new PluginImplementationError('Realtime controller is not available in plugins. You should use plugin hooks instead');
+        },
+        security: sdk.security,
+        server: sdk.server
+      });
+
+      // TODO: Wait for the end of Node.js 6 support to use the spread operator
       Object.defineProperty(this.accessors, 'sdk', {
         enumerable: true,
         get: () => ({
-          query: sdk.query.bind(sdk),
-          auth: sdk.auth,
-          bulk: sdk.bulk,
-          collection: sdk.collection,
-          document: sdk.document,
-          index: sdk.index,
-          ms: sdk.ms,
-          realtime: sdk.realtime,
-          security: sdk.security,
-          server: sdk.server
+          query: kuzzleSdk.query.bind(kuzzleSdk),
```

Comment:
Why don't you use your new `getPluginSdk` method to prevent duplicated code?

Something like:

```js
get: () => {
  const wrapped = getPluginSdk(kuzzleSdk);
  wrapped.as = /* your "as" method is fine */

  return wrapped;
}
```